workflow-dev/tpl/deis-controller-rc.yaml

```diff
@@ -53,13 +53,13 @@ spec:
                 secretKeyRef:
                   name: builder-key-auth
                   key: builder-key
-{{- if or (eq (env "DATABASE_LOCATION") "off-cluster") (eq .database_location "off-cluster") }}
+{{- if env "DATABASE_LOCATION" | default .database_location | eq "off-cluster" }}
```

Comment:
This corrects it by defaulting to what `.database_location` says _only_ if the `DATABASE_LOCATION ` env var is not defined.